gecko_sdk_extensions/nc_efr32_watchdog_extension/nc_efr32_watchdog.slce

```diff
@@ -0,0 +1,8 @@
+id: "nc_efr32_watchdog"
+label: "EFR32 Watchdog"
+version: "1.0.0"
+sdk:
+  id: "gecko_sdk"
+  version: 4.3.2
```

Comment:
I think its good "´ticking" the version then using one old version of it that its not working as intended and logging it in the addon startup.

Is it locked to only being used with GSDK 4.3.2 ? or its only using it as code base from the 4.3.2?

Reply:
It'll be a little difficult to log something from the firmware itself, as we would need to send some sort of message over CPC and have the addon receive it.

The watchdog is implemented as a proper SDK extension so it is compatible with any Gecko SDK version. The next watchdog unfortunately won't be, as there is no way to hook into CPC without editing the SDK.